test/OrchardCore.Tests/DisplayManagement/LiquidTests.cs

```diff
@@ -0,0 +1,127 @@
+using System.Text.Json;
+using Fluid;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.Liquid;
+using OrchardCore.Tests.Apis.Context;
+
+namespace OrchardCore.Tests.DisplayManagement
+{
+    public class LiquidTests
+    {
+        [Fact]
+        public async Task ComparingTextField_ReturnsCorrectValue()
+        {
+            var context = new SiteContext();
+            await context.InitializeAsync();
+            await context.UsingTenantScopeAsync(async scope =>
+            {
+                var template = """
+                    {% if Model.ContentItem.Content.MyPart.myField.Text == "Some test value" %}true{% else %}false{% endif %}
+                    """;
+
```

Comment:
```suggestion
```

Reply:
Let's keep this line for better readability.